src/common/servlist.c

```diff
@@ -58,6 +58,9 @@ static const struct defaultserver def[] =
 	/* Self signed */
 	{0,			"irc.accessirc.net"},
 
+	{"ACN", 0},
```

Comment:
Update this to include SASL and SSL like other entries.

Reply:
Done